src/commands/dev/edge/watch.rs

```diff
@@ -51,11 +51,14 @@ pub fn watch_for_changes(
                     if err.0.contains("10049") {
                         refresh_session_channel.send(Some(()))?;
                         break;
+                    } else if err.0.contains("10021") {
+                        // Continue to watch for changes if it's a syntax error
+                        StdOut::warn(&format!("{}\nPlease update the your code.", &err.0));
```

Comment:
Is the additional context necessary? It would be helpful to give an example here of what the upstream error looks like.

Either way, this isn't grammatical.
```suggestion
                        StdOut::warn(&format!("{}\nPlease update your code.", &err.0));
```

Reply:
After formatting, the error looks like this:
```
⚠️  Something went wrong with the request to Cloudflare...
Uncaught SyntaxError: Unexpected token 'return'
  at line 9
 [API code: 10021]
```